tests/src/Kernel/Paragraphs/LinksBlockTest.php

```diff
@@ -0,0 +1,360 @@
+
+    $html = $this->renderParagraph($paragraph);
+    $crawler = new Crawler($html);
+
+    $this->assertBackgroundTransparent($crawler);
+    $this->assertLinksBlockRendering($crawler);
+    $this->assertHorizontalLinks($crawler, FALSE);
+  }
+
+  /**
+   * Returns a list of link items for Social media follow paragraph.
+   *
+   * @return array
+   *   An array of link items.
+   */
+  protected function getSocialMediaLinks(): array {
+    return [
+      [
+        'title' => 'Email',
+        'uri' => 'mailto:example@com',
+        'link_type' => 'email',
+      ],
+      [
+        'title' => 'Facebook',
+        'uri' => 'https://facebook.com',
+        'link_type' => 'facebook',
+      ],
+      [
+        'title' => 'Flickr',
+        'uri' => 'https://www.flickr.com',
+        'link_type' => 'flickr',
+      ],
+      [
+        'title' => 'Google+',
+        'uri' => 'https://google.com',
+        'link_type' => 'google',
+      ],
+      [
+        'title' => 'Instagram',
+        'uri' => 'https://instagram.com',
+        'link_type' => 'instagram',
+      ],
+      [
+        'title' => 'LinkedIn',
+        'uri' => 'https://linkedin.com',
+        'link_type' => 'linkedin',
+      ],
+      [
+        'title' => 'Pinterest',
+        'uri' => 'https://pinterest.com',
+        'link_type' => 'pinterest',
+      ],
+      [
+        'title' => 'RSS',
+        'uri' => 'https://rss-example.com',
+        'link_type' => 'rss',
+      ],
+      [
+        'title' => 'Storify',
+        'uri' => 'https://storify.com',
+        'link_type' => 'storify',
+      ],
+      [
+        'title' => '1st Twitter',
+        'uri' => 'https://twitter.com',
+        'link_type' => 'twitter',
+      ],
+      [
+        'title' => '2nd Twitter',
+        'uri' => 'https://twitter.com',
+        'link_type' => 'twitter',
+      ],
+      [
+        'title' => 'Yammer',
+        'uri' => 'https://yammer.com',
+        'link_type' => 'yammer',
+      ],
+      [
+        'title' => 'Youtube',
+        'uri' => 'https://youtube.com',
+        'link_type' => 'youtube',
+      ],
+      [
+        'title' => 'Vimeo',
+        'uri' => 'https://vimeo.com',
+        'link_type' => 'vimeo',
+      ],
+    ];
+  }
+
+  /**
+   * Returns a list of link items for Block Links paragraph.
+   *
+   * @return array
+   *   An array of link items.
+   */
+  protected function getBlockLinks(): array {
+    return [
+      [
+        'title' => 'European Commission',
+        'uri' => 'https://example.com',
+      ],
+      [
+        'title' => 'Priorities',
+        'uri' => 'https://example.com',
+      ],
+      [
+        'title' => 'Jobs, Growth and Investment',
+        'uri' => 'https://example.com',
+      ],
+    ];
+  }
+
+  /**
+   * Assert links from social media.
+   *
+   * @param \Symfony\Component\DomCrawler\Crawler $crawler
+   *   The DomCrawler where to check the element.
+   */
+  private function assertSocialLinks(Crawler $crawler): void {
```

Comment:
its sufficient to make these protected, phpunit will ignore them and this way any class extending this test class could still benefit from these methods